wolfcrypt/test/test.c

```diff
@@ -6392,8 +6400,13 @@ int ecc_test(void)
 
 #ifdef HAVE_ECC_SIGN
     /* test DSA sign hash */
-    for (i = 0; i < (int)sizeof(digest); i++)
+    for (i = 0; i < (int)sizeof(digest); i++) {
+    #ifdef ECC_TEST_VERIFY_ZERO
```

Comment:
The change looks good, and fixes the issue. With testing does this eliminate the previous test with a digest of 0123.....  when FP_ECC is defined? If so I think we should broaden the test coverage and still test the previous case in addition to testing a digest of all 0's rather then narrowing the test coverage. This is not critical though and can be a future change if wanted.